assets/src/design-system/components/snackbar/snackbarMessage.js

```diff
@@ -121,29 +145,39 @@ const SnackbarMessage = ({
   }, []);
 
   return (
-    <MessageContainer
-      role="alert"
-      aria-label={ariaLabel}
-      hasAction={Boolean(actionLabel)}
-    >
+    <MessageContainer role="alert" hasAction={Boolean(actionLabel)} {...props}>
```

Comment:
allows us to require `'aria-label'` and we can use styled components here since we can pass a classname to the outer container now.